### DISCUSSION TITLE
How can I delete a parameter sweep variation with pyaedt?

### DISCUSSION THREAD

**Discussion by RyanAshbaugh** [ansys/pyaedt#3763] [Q&A] [answered]
I'm wondering how I can delete a parameter sweep operation in Maxwell3d. I have added multiple variables to my sweep, each switching between a single value.

Say I have three variables, var1, var2 and var3, and I want them each to take on values 0 and 1.

I use something like the below to add it:

            m3d.parametrics.add(
                "var1",
                0,
                parametricname=parametric_name,
                variation_type="SingleValue",
            )

and just repeat for each variable and value.

Using the regular Python scripting function in AEDT I was able to do something like the following

```
oModule.InsertSetup("OptiParametric",
	[
		"NAME:ParametricSetup1",
		"IsEnabled:="		, True,
		[
			"NAME:ProdOptiSetupDataV2",
			"SaveFields:="		, True,
			"CopyMesh:="		, True,
			"SolveWithCopiedMeshOnly:=", True
		],
		[
			"NAME:StartingPoint"
		],
		"Sim. Setups:="		, ["Setup1"],
		[
			"NAME:Sweeps",
			[
				"NAME:SweepDefinition",
				"Variable:="		, "$var1",
				"Data:="		, "0 1",
				"OffsetF1:="		, False,
				"Synchronize:="		, 0
			],
			[
				"NAME:SweepDefinition",
				"Variable:="		, "$var2",
				"Data:="		, "0 1",
				"OffsetF1:="		, False,
				"Synchronize:="		, 0
			],
			[
				"NAME:SweepDefinition",
				"Variable:="		, "$var3",
				"Data:="		, "0 1",
				"OffsetF1:="		, False,
				"Synchronize:="		, 0
			]
		],
		[
			"NAME:Sweep Operations",
			"del:="			, ["1","1","1"],
			"del:="			, ["1","1","0"],
			"del:="			, ["1","0","1"],
			"del:="			, ["0","1","1"]
		],
		[
			"NAME:Goals"
		]
	])
```

How can I do something similar with pyaedt and delete the sweeps that I don't want?

**Comment by Samuelopez-ansys** [comment, marked as answer]
@RyanAshbaugh It is not possible, please submit an issue if you need this new feature.
